comps/guardrails/pii_detection/README.md

```diff
@@ -1,6 +1,23 @@
 # PII Detection Microservice
 
-PII Detection a method to detect Personal Identifiable Information in text. This microservice provides users a unified API to either upload your files or send a list of text, and return with a list following original sequence of labels marking if it contains PII or not.
+This microservice provides a unified API to detect if there is Personal Identifiable Information or Business Sensitive Information in text. Users can send a list of files, a list of text strings, or a list of urls to the microservice, and the microservice will return a list of True or False for each piece of text following the original sequence.
+
+We provide 2 detection strategies:
```

Comment:
There are three strategies in the code, but only two were mentioned in the readme

Reply:
The 3rd strategy LLM based detection is not implemented currently, but we plan to add later in future.

Reply:
Deleted LLM strategy related code.